sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -92,6 +92,12 @@ public partial class AzurePowerShellCredentialOptions : Azure.Identity.TokenCred
         public System.TimeSpan? ProcessTimeout { get { throw null; } set { } }
         public string TenantId { get { throw null; } set { } }
     }
+    public partial class BrowserCustomizationOptions
+    {
+        public BrowserCustomizationOptions() { }
+        public BrowserCustomizationOptions(string title) { }
+        public BrowserCustomizationOptions(System.Uri browserRedirectError, System.Uri browserRedirectSuccess, string htmlMessageSuccess, string htmlMessageError, bool iOSHidePrivacyPrompt, System.Func<System.Uri, System.Threading.Tasks.Task> openBrowserAsync) { }
```

Comment:
> We should expose the specific properties we intend to support rather than only exposing properties via the ctor:

Do you think the version below would be better?
```
    public class WebViewCustomizationOptions
    {
        public bool? UseEmbeddedWebView { get; set; }

        internal SystemWebViewOptions SystemBrowserOptions;

        internal EmbeddedWebViewOptions EmbeddedBrowserOptions;

        private SystemWebViewOptions systemWebViewOptions
        {
            get
            {
                if (SystemBrowserOptions == null)
                {
                    SystemBrowserOptions = new SystemWebViewOptions();
                }
                return SystemBrowserOptions;
            }
        }

        private EmbeddedWebViewOptions embeddedWebViewOptions
        {
            get
            {
                if (EmbeddedBrowserOptions == null)
                {
                    EmbeddedBrowserOptions = new EmbeddedWebViewOptions();
                }
                return EmbeddedBrowserOptions;
            }
        }

        public Uri BrowserRedirectError
        {
            set
            {
                systemWebViewOptions.BrowserRedirectError = value;
            }
        }
        public Uri BrowserRedirectSuccess
        {
            set
            {
                systemWebViewOptions.BrowserRedirectSuccess = value;
            }
        }
        public string HtmlMessageError
        {
            set
            {
                systemWebViewOptions.HtmlMessageError = value;
            }
        }
        public string HtmlMessageError
        {
            set
            {
                systemWebViewOptions.HtmlMessageError = value;
            }
        }
        public bool iOSHidePrivacyPrompt
        {
            set
            {
                systemWebViewOptions.iOSHidePrivacyPrompt = value;
            }
        }

        public Func<Uri, System.Threading.Tasks.Task> OpenBrowserAsync
        {
            set
            {
                systemWebViewOptions.OpenBrowserAsync = value;
            }
        }
        public string Title
        {
            set
            {
                embeddedWebViewOptions.Title = value;
            }
        }

        public WebViewCustomizationOptions()
        {
            UseEmbeddedWebView = null;
            SystemBrowserOptions = null;
            EmbeddedBrowserOptions = null;
        }
    }
```

Reply:
Looks better - I would eliminate the properties that are not strictly required for your scenario though. 
e.g. 
`iOSHidePrivacyPrompt`
`OpenBrowserAsync`

Reply:
> We should expose the specific properties we intend to support rather than only exposing properties via the ctor:

Sorry for late response. I was working to integrating the new interface to implement our new feature to convince of the interfaces we depended on.

Currently, we have some change on the interfaces we depended on.
What we needed is  `HtmlMessageSuccess` and `HtmlMessageError`.
And so I revised the PR accordingly. Please review it.